### DISCUSSION TITLE
Old plots are not recognized

### DISCUSSION THREAD

**Discussion by kastermight** [Chia-Network/chia-blockchain#6190] [Support]
Hi everyone. I'm having an issue with my plots. I started plotting and farming on a single machine by creating a wallet and associated master/farm/pool keys. Then at some point I got another machine that I used as a plotter and set up a harvester (per official instructions, copying ssl, using farm and pool keys etc) on that machine. Everything was working fine for some time. Then, I got a JBOD that I put hard drives with all the plots from **both** machines to that enclosure and connected it to the harvester. Since my harvester is much more powerful than my first machine (full node) I wanted to switch their places. So I used same wallet id (using 24 word mnemonics) and manually added all those hard drives to the config file, and ran a full node on the second (former harvester) machine. But those plots are not recognized anymore. It's not like I see any errors or anything, but when I do 

`chia farm summary`

it shows 0 plots. I have around 500 plots on those hard drives combined. Please help.

Thanks.

**Comment by kastermight** [comment]
I forgot to change farmer's ip address. Figured it out with the help from discord. Case closed :)

**Comment by BrandtH22** [comment]
@kastermight , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
